docs/tutorial/native-file-drag-drop.md

```diff
@@ -14,28 +14,47 @@ API in response to the `ondragstart` event.
 
 ## Example
 
-Starting with a working application from the
-[Quick Start Guide](quick-start.md), add the following lines to the
-`index.html` file:
+A simple example demonstrating how you can create a file on the fly to be dragged out of the window.
```

Comment:
```suggestion
An example demonstrating how you can create a file on the fly to be dragged out of the window.
```

I think we should not use terms like "simple", "easy", etc. in the documentation. People might understand what's going on at different speeds and if they need to read the docs the task at hand might not be easy already.

Jim Fischer gave a talk a few years back about this: [Don't say "simply"](https://www.youtube.com/watch?v=gsT2BBWBVmM).